lte/gateway/python/magma/subscriberdb/store/sqlite.py

```diff
@@ -264,6 +269,14 @@ def get_subscriber_data(self, subscriber_id):
                 row = res.fetchone()
                 if not row:
                     raise SubscriberNotFoundError(subscriber_id)
+        except sqlite3.OperationalError:
+            # Print the process holding the lock
+            db_parts = db_location.split(":", 1)
+            if (len(db_parts) == 2) and db_parts[1]:
+                path_str = db_parts[1].split("?")
+                import os
+                os.system('fuser -uv ' + path_str[0])
```

Comment:
update this to `subprocess.call()` or `Popen()`, retrieve the stdout and then log it using `logging` library

Reply:
Comments addressed.